kong/core/handler.lua

```diff
@@ -696,17 +696,20 @@ return {
       local header = ngx.header
 
       if ctx.KONG_PROXIED then
-        if singletons.configuration.latency_tokens then
+        if singletons.configuration.headers[constants.HEADERS.UPSTREAM_LATENCY] then
           header[constants.HEADERS.UPSTREAM_LATENCY] = ctx.KONG_WAITING_TIME
+        end
+
+        if singletons.configuration.headers[constants.HEADERS.PROXY_LATENCY] then
```

Comment:
style: the line below has now has a weird looking assignment operator with too many spaces, fixing it should be in the scope of this PR